modules/native_streaming_client_module/src/native_streaming_client_module_impl.cpp

```diff
@@ -307,8 +316,11 @@ PropertyObjectPtr NativeStreamingClientModule::populateDefaultConfig(const Prope
                 for (const auto& transportLayerProp : defTransportLayerConfig.getAllProperties())
                 {
                     const auto transportLayerName = transportLayerProp.getName();
-                    if (transportLayerConfig.hasProperty(name))
-                        defTransportLayerConfig.setPropertyValue(name, config.getPropertyValue(name));
+                    if (transportLayerConfig.hasProperty(transportLayerName))
```

Comment:
This nested populate of TransportLayerConfig could be extracted into a separate method.